### DISCUSSION TITLE
How complete is klippa

### DISCUSSION THREAD

**Discussion by MIRIMIRIM** [googlefonts/fontations#985] [Q&A] [answered]
I’m curious about the level of completion of Klippa. I tried to use it to create a subset font, but found that the subset font could not be used normally.

**Comment by garretrieger** [comment, marked as answer]
Klippa is still extremely early in development and is currently incomplete. In the mean time you could consider using the [harfbuzz subsetting](https://github.com/harfbuzz/harfbuzz/blob/main/src/hb-subset.h) library/client which is complete.
